python/py-spdx-lookup/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-spdx-lookup
+version             0.3.2
+revision            0
+
+maintainers         nomaintainer
+description         SPDX license list query tool
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/bbqsrc/spdx-lookup-python
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
```

Comment:
This is just a random rant and doesn't mean that you need to do anything about it.

This is one case where one could make an improvement in package automation, but I'm not entirely sure how precisely. There are two major ways to fetch pypi packages. One can either fetch from pypi, or from github directly. To me (but that's a 100% personal opinion, not one of other maintainers) it looks slightly cleaner when one is fetching via github PortGroup. But even when fetching from pypi, we might perhaps improve support for the ugly line specifying `master_sites`. That line could be automated with some fixes in the PortGroup.

Reply:
`pypi:s/spdx-lookup` can be easily done with upt as suggested by @reneeotten but I avoided doing it seeing the convention in other existing Ports.

If in future theres a plan to add many pypi ports then it would make sense to just write a port group for the same, making the port look more clean.

Also for pypi ports setup tools is I guess a compulsion soo even that could be added to the portgroup.

Reply:
I believe https://trac.macports.org/ticket/56642 is related.

Reply:
I am not sure how portgroups work but these two lines are only there for pypi ports.
Won't it cause an issue to include these two lines for non-pypi python ports?